core/pom.xml

```diff
@@ -408,6 +372,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <configuration>
+          <shadedArtifactAttached>false</shadedArtifactAttached>
+          <artifactSet>
+            <includes>
+              <include>com.google.guava:guava</include>
+              <include>org.apache.spark:spark-network-common_${scala.binary.version}</include>
+              <include>org.apache.spark:spark-network-shuffle_${scala.binary.version}</include>
```

Comment:
Is this going to inline other modules (spark-network-common and spark-network-shuffle) in the published spark-core jar?